nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenHTTP.java

```diff
@@ -159,11 +160,20 @@ public AllowableValue getAllowableValue() {
             .build();
     public static final PropertyDescriptor AUTHORIZED_DN_PATTERN = new PropertyDescriptor.Builder()
         .name("Authorized DN Pattern")
-        .description("A Regular Expression to apply against the Distinguished Name of incoming connections. If the Pattern does not match the DN, the connection will be refused.")
+        .description("A Regular Expression to apply against the Subject's Distinguished Name of incoming connections. If the Pattern does not match the Subject DN, " +
```

Comment:
I would add "Subject" in the property name on the UI to make the Subject / Issuer distinction more clear. It can be achieved via adding `displayName` with value "Authorized Subject DN Pattern".
(The existing `name` cannot be changed due to backward compatibility.)